runtime/databricks/automl_runtime/forecast/pmdarima/model.py

```diff
@@ -0,0 +1,323 @@
+#
+# Copyright (C) 2021 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from typing import List
+
+import pandas as pd
+import mlflow
+import pmdarima
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+
+
+class ArimaModel(mlflow.pyfunc.PythonModel):
+    """
+    ARIMA mlflow model wrapper for univariate forecasting.
+    """
+
+    def __init__(self, pickled_model: bytes, horizon: int, frequency: str,
+                 start_ds: pd.Timestamp, end_ds: pd.Timestamp, time_col: str) -> None:
+        """
+        Initialize the mlflow Python model wrapper for ARIMA.
+        :param pickled_model: the pickled ARIMA model as a bytes object.
+        :param horizon: int number of periods to forecast forward.
+        :param frequency: the frequency of the time series
+        :param start_ds: the start time of training data
+        :param end_ds: the end time of training data
+        :param time_col: the column name of the time column
+        """
+        self._pickled_model = pickled_model
+        self._horizon = horizon
+        self._frequency = frequency
+        self._start_ds = start_ds
+        self._end_ds = end_ds
+        self._time_col = time_col
+        super().__init__()
```

Comment:
Nit: prefer to call `super().__init__()` at the beginning, unless you need some do some preprocessing and pass the result to `super().__init__()`.